persistence/pom.xml

```diff
@@ -16,5 +16,15 @@
       <artifactId>model</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
we will not need these dependencies if I can implement the transactions the simple way through the `@Transactional` annotation.